miio/integrations/zhimi/airpurifier/tests/test_airpurifier_miot.py

```diff
@@ -31,8 +31,10 @@
     "filter_rfid_product_id": "0:0:41:30",
     "filter_rfid_tag": "10:20:30:40:50:60:7",
     "button_pressed": "power",
+    "aqi_realtime_update_duration": 0,
```

Comment:
The default value for my 3H for this setting is 0. I can move this change to device specific init state if that is desired.